### DISCUSSION TITLE
Email report

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#845] [Q&A]
Hello,
please, is it possible to download reports from the HMI environment?
Not by email.

Thank you

**Comment by unocelli** [comment]
![image](https://github.com/frangoteam/FUXA/assets/48684550/45233393-cf78-4c59-9923-031e18d06f32)

**Reply by PaulRohlik** [reply]
Thank you very much for your answer.
I know this possibility. As an admin I can download the report from the editor. But I can't let the operator into the editor. That's why I wonder if it can be done in some way from hmi space
